src/http/api/image-upload.ts

```diff
@@ -20,7 +17,9 @@ export function createStorageHandler(
         }
 
         const account = await accountService.getAccountForSite(ctx.get('site'));
-        const result = await storageService.saveFile(file, account.uuid);
+        const path = imageStorageService.storagePath(file, account.uuid);
```

Comment:
NABD - I would probably even put this inside `save` tbh

Reply:
Updated in 7944aaa22a71a5d48d1561b49371fdd06d7ae83c